pages/glossary/included-in-the-accessibility-tree.md

```diff
@@ -11,45 +11,6 @@ Elements included in the accessibility tree of platform specific accessibility A
-
-#### Examples
-
-**Note:** The examples presented here are non-normative and not testable. They serve to illustrate some common pitfalls about the definition and to help implementers of ACT rules understand it.
-
-This `span` element is included in the accessibility tree (by default, elements are included in the accessibility tree).
-
-```html
-<span>ACT rules</span>
-```
-
-This `span` element is not included in the accessibility tree because it is hidden to everybody by the CSS property.
-
-```html
-<span style="display:none">ACT rules</span>
-```
-
-This `span` element is not included in the accessibility tree because it is explicitly removed by the `aria-hidden` attribute.
+For more details, see [examples of included in the accessibility tree](/pages/examples/included-in-the-accessibility-tree/).
 
-```html
-<span aria-hidden="true">ACT rules</span>
-```
-
-This `span` element is positioned off screen, hence is not [visible](#visible), but is nonetheless included in the accessibility tree.
-
-```html
-<span style="position: absolute; top:-9999em">ACT rules</span>
-```
-
-Although the `span` element with an `id` of "label" is not itself included in the accessibility tree, it still provides an [accessible name](#accessible-name) to the other `span`, via the `aria-labelledby` attribute. Thus, it is still indirectly exposed to users of assistive technologies. Removing an element from the accessibility tree is not enough to remove all accessibility concerns from it since it can still be indirectly exposed.
-
-```html
-<span id="label" style="display:none">ACT rules</span>
-<span aria-labelledby="label">Accessibility Conformance Testing rules</span>
-```
-
-Although this `input` element is not included in the accessibility tree, it is still [focusable](#focusable), hence users of assistive technologies can still interact with it by sequential keyboard navigation. This may result in confusing situations for such users (and is in direct violation of [the fourth rule of ARIA (working draft)](https://www.w3.org/TR/using-aria/#fourth)).
-
-```html
-<input type="text" aria-hidden="true" name="fname" />
-```
+**Note:** Users of assistive technologies might still be able to interact with elements that are not included in the accessibility tree. An example of this is a [focusable](#focusable) element with an `aria-hidden` attribute with a value of `true`. Such an element could still be interacted with using sequential keyboard navigation regardless of the assistive technologies used, even though the element would not be included in the accessibility tree.
```

Comment:
Plus, technically, "focusable" does not imply "in sequential keyboard navigation order", so the text is not fully technically correct, but I say it's OK to keep it that way in this context (example in a note).